site/pages/institutional-service-performance-en.hbs

```diff
@@ -12,14 +12,14 @@
 ---
 {{>byline}}
 <p>This report covers the 2015 to 2016 fiscal year. A fiscal year runs from April 1 to March 31.</p>
-<section>
+<nav role="navigation">
```

Comment:
Is it required to duplicate the role? By definition the [```<nav>``` element](https://www.w3.org/TR/html52/sections.html#ref-for-represent-13) imply the aria role navigation.

Idem for all other instance.

Reply:
Check out https://github.com/wet-boew/wet-boew/pull/8177#discussion_r146637014 for my response to this.